index.html

```diff
@@ -77,6 +77,39 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
 
+
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
You need to rename `*TEMPLATE: MAKE A COPY* Contributor`
into `your_name`
Otherwise, someone reading the code, will not be able to identify your card.